python/ray/air/_internal/usage.py

```diff
@@ -115,3 +118,40 @@ def tag_scheduler(scheduler: "TrialScheduler"):
     assert isinstance(scheduler, TrialScheduler)
     scheduler_name = _find_class_name(scheduler, "ray.tune.schedulers", TUNE_SCHEDULERS)
     record_extra_usage_tag(TagKey.TUNE_SCHEDULER, scheduler_name)
+
+
+def tag_multinode_cluster_storage(
+    local_path: str, remote_path: Optional[str], sync_config: "SyncConfig"
+) -> bool:
+    import ray
+
+    if len(ray.nodes()) <= 1:
+        return False
+
+    nfs = local_path and not remote_path and os.path.ismount(local_path)
```

Comment:
Does `os.path.ismount` generically determine if something is NFS?

Reply:
Need to do a bit more research and testing on this!